README.md

```diff
@@ -1,26 +1,190 @@
 
-```bash
-pip install sfctl
+Following table gives the mapping of sfctl versions and Service Fabric runtime versions.
+<table>
+<thead>
+<tr>
+<th>sfctl version</th><th>Service Fabric runtime version</th>
+</tr>
+</thead>
+<tr><td>1.1.0</td><td>5.6.130 or later</td></tr>
+<tr><td>2.0.0</td><td>6.0 and later</td></tr>
+</table>
+	
+## Installing sfctl using pip
+
+#### Perquisite
+[Python 3.6](https://www.python.org/) or later installed on the machine.
+
+pip commands to Install, Upgrade and Uninstall to a central location on the machine require running commands from an admin Command prompt (Windows) or a Terminal window with sudo access (Linux or Mac). On Linux or Mac, prepend sudo to each of these commands below.
+
+#### Install latest version
+```
+pip3 install sfctl
+```
+#### Install a specific version
+Identify the the most appropriate version from table above and specify it in the following command
+```
+pip3 install sfctl==replace-with-required-version
+```
+Following will install version 1.1.0 of sfctl
+```
+pip3 install sfctl==1.1.0
+```
+#### Determine version of sfctl installed on the machine
 ```
+pip3 show sfctl
+```
+#### Upgrade to latest version
+```
+pip3 install -U sfctl
+```
+#### Upgrade to specific version
+```
+pip3 install -U sfctl==replace-with-required-version
+```
+#### Uninstall
+```
+pip uninstall -y sfctl
+```
+## Installing multiple versions side-by-side using python virtual environment
+There may be a need to install different versions of sfctl side by side to interact with clusters running different versions of Service Fabric runtime. 
+
+[pipenv](https://docs.pipenv.org/) package can be used to setup a python virtual environment that can be used to install a different version of sfctl separate from the globally installed version.
 
-### Launching
+1. Install pipenv.
 
-```bash
-sfctl <command>
+    **NOTE:** Only this command requires an admin command prompt or sudo access. Do not use admin command prompt or sudo for rest of the commands below
+    ```
+    pip3 install pipenv
+    ```
+2. Create a directory for installing files for virtual environment (commands below installs sfctl 1.1.0)
+    ```
+    mkdir sfctl-1.1.0
+    ```
+3. Install sfctl in virtual env
+    ```
+    pipenv --three install sfctl-1.1.0
+    ```
+4. Create a shell that uses files in virtual environment
+    ```
+    pipenv --three shell
+    ```
+5. Run 1.1.0 of sfctl
+    ```
+    pip show sfctl
+    sfctl --help
+    ```
+6. Exit virtual environment shell
+    ```
+    exit
+    ```
+7. Run globally installed version of sfctl
+    ```
+    pip show sfctl
+    sfctl --help
+    ```
+
+## Installing multiple versions side-by-side in a Docker container
+sfctl github repo has Dockerfile to generate a container with the required version of sfctl.

```

Comment:
There's already documentation in the Wiki tab on how to use `virtualenv` to run multiple versions of sfctl. Using docker images is too heavy for just this reason